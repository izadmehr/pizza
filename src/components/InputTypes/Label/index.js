import styled from "styled-components";

const Label = styled.label`
  display: block;
  font-weight: bolder;
  padding: 0;
  margin-bottom: 20px;

  input,
  textarea {
    display: block;
  }
`;

export default Label;
