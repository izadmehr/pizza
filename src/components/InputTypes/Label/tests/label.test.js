import React from "react";
import { mount } from "enzyme";

import "jsdom-global/register";
import "jest-styled-components";
import Label from "../index";

describe("Label Component", () => {
  it("has correct default styles", () => {
    const wrapper = mount(<Label>Test Label</Label>);
    expect(wrapper).toMatchSnapshot();
  });
});
