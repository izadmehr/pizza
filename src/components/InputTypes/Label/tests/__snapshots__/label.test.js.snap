// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label Component has correct default styles 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-weight: bolder;
  padding: 0;
}

.c0 input,
.c0 textarea {
  margin-right: 0.5rem;
}

<styled.label>
  <StyledComponent
    forwardedClass={
      Object {
        "$$typeof": Symbol(react.forward_ref),
        "attrs": Array [],
        "componentStyle": ComponentStyle {
          "componentId": "sc-bdVaJa",
          "isStatic": true,
          "lastClassName": "c0",
          "rules": Array [
            "
  display: flex;
  align-items: center;
  font-weight: bolder;
  padding: 0;

  input,
  textarea {
    margin-right: 0.5rem;
  }
",
          ],
        },
        "displayName": "styled.label",
        "render": [Function],
        "styledComponentId": "sc-bdVaJa",
        "target": "label",
        "toString": [Function],
        "warnTooManyClasses": [Function],
        "withComponent": [Function],
      }
    }
    forwardedRef={null}
  >
    <label
      className="c0"
    >
      Test Label
    </label>
  </StyledComponent>
</styled.label>
`;
